components/compositing/tracing.rs

```diff
@@ -54,6 +54,7 @@ mod from_constellation {
                 Self::DelayNewFrameForCanvas(..) => target!("DelayFramesForCanvas"),
                 Self::ScreenshotReadinessReponse(..) => target!("ScreenshotReadinessResponse"),
                 Self::SendLCPCandidate(..) => target!("SendLCPCandidate"),
+                Self::LoadUrl(..) => target!("LoadUrl"),
```

Comment:
Let's make this message more general. Call it `Self::ReenableLargestContentfulPaintCalculation`.